modules/apps/27-interchain-accounts/controller/ibc_middleware.go

```diff
@@ -245,7 +245,8 @@ func (IBCMiddleware) OnChanUpgradeAck(ctx sdk.Context, portID, channelID, counte
 }
 
 // OnChanUpgradeOpen implements the IBCModule interface
-func (IBCMiddleware) OnChanUpgradeOpen(ctx sdk.Context, portID, channelID string) {}
+func (IBCMiddleware) OnChanUpgradeOpen(ctx sdk.Context, portID, channelID string, order channeltypes.Order, connectionHops []string, version string) {
+}
```

Comment:
it bothers me that this is on a separate line when the other ones aren't, but the linter is happy with this so 🤷

Reply:
yeah for some reason my editor kept doing this on save :D